portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -85,17 +87,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         String author = getParameter(request, "author", "unknown");
         String text = getParameter(request, "text", "");
+        String page = getParameter(request, "page", "index.html");
```

Comment:
I am unsure about the default value of `"index.html"`. It seems that there is no comment section on the main page anymore. So if we have a bad request with no "page" parameter, the comment will get stored but will never be viewed or deleted.

If we still want to store a bad request instead of dropping it maybe "unknown" is a better default value?